components/sql/src/main/scala/pl/touk/nussknacker/sql/db/query/QueryResultStrategy.scala

```diff
@@ -31,3 +33,8 @@ case object ResultSetStrategy extends QueryResultStrategy {
 case object SingleResultStrategy extends QueryResultStrategy {
   val name: String = "Single result"
 }
+
+case object UpdateResultStrategy extends QueryResultStrategy {
+  type updateResultType = Integer
+  val name: String = "Update result"
```

Comment:
I think that this description is meaningless to user. WDYT about renaming it to `Updates count` or `Updated records count` or `Affected records count`?